#goal definition
bool startMotor
---
#result definition
bool accept
---
#feedback
bool isMotorOn
